website/content/docs/ecs/architecture.mdx

```diff
@@ -9,37 +9,51 @@ description: >-
 
 ![Consul on ECS Architecture](/img/consul-ecs-arch.png)
 
-As shown above there are two main components to the architecture.
+As shown above, these are the main components to the architecture for a secure installation:
 
-1. **Consul Server task:** Runs the Consul server.
+1. **Consul Servers:** Production-ready Consul server cluster
 1. **Application tasks:** Runs user application containers along with two helper containers:
    1. **Consul Client:** The Consul client container runs Consul. The Consul client communicates
       with the Consul server and configures the Envoy proxy sidecar. This communication
       is called _control plane_ communication.
    1. **Sidecar Proxy:** The sidecar proxy container runs [Envoy](https://envoyproxy.io/). All requests
       to and from the application container(s) run through the sidecar proxy. This communication
       is called _data plane_ communication.
+1. **ACL Controller:** Automatically provisions Consul ACL tokens for Consul clients and service mesh services
+   in an ECS Cluster.
 
 For more information about how Consul works in general, see Consul's [Architecture Overview](/docs/architecture).
 
-In addition to the long-running Consul Client and Sidecar Proxy containers, there
-are also two initialization containers that run:
-
-1. `discover-servers`: This container runs at startup and uses the AWS API to determine the IP address of the Consul server task.
-1. `mesh-init`: This container runs at startup and sets up initial configuration for Consul and Envoy.
+In addition to the long-running Consul Client and Sidecar Proxy containers, the `mesh-init` container runs
```

Comment:
```suggestion
In addition to the long-running Consul client and sidecar proxy containers, the `mesh-init` container runs
```
I think this is more consistent with our style guide--at least, other references use lower case. But ignore if I'm mistaken.